src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/aos/AcknowledgementOfClaimGenerator.java

```diff
@@ -92,7 +92,7 @@ public AcknowledgementOfClaimForm getTemplateData(CaseData caseData) {
                                       .primaryAddress(respondent2.getPrimaryAddress())
                                       .representative(representativeService.getRespondent2Representative(caseData))
                                       .litigationFriendName(
-                                          ofNullable(caseData.getRespondent1LitigationFriend())
+                                          ofNullable(caseData.getRespondent2LitigationFriend())
```

Comment:
Could we add a unit test for checking this data is returned via the callback (specific to respondent2) - it was probably missed first time round.
I can see there is an example for respondent one, checking the `expectedDocmosisData`

Reply:
added expectedDocmosisData for 1v2 cases